components/foodcritic-wrapper/go.sum

```diff
@@ -0,0 +1,128 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
Is there a way to do this without these large wrappers. This wrapper is going to be bigger than Foodcritic and its deps